ChiselTests/CHLDuplicateDetectionTests.m

```diff
@@ -0,0 +1,68 @@
+//
+//  CHLDuplicateDetectionTests.m
+//  Chisel
+//
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "CHLDuplicateDetection.h"
+
+@interface _CHLTestClass : NSObject
+@property (nonatomic, strong) NSString *identifier;
+@end
+@implementation _CHLTestClass
+@end
+
+@interface CHLDuplicateDetectionTests : XCTestCase
+@end
+
+@implementation CHLDuplicateDetectionTests
+
+- (void)testDuplicateDetectionCanFindTwoCustomObjects
+{
+  @autoreleasepool {
```

Comment:
Oh I should clean it up, in the ideal world I thought it might be useful but also - the idea changed, not to mention these objects are retain cycles, so they get no releases :P